### DISCUSSION TITLE
Color Theming

### DISCUSSION THREAD

**Discussion by fiumad** [usememos/memos#1216] [Ideas]
Hello all!

I am really interested in adding color theming to this project. I think Memos could really benefit from having great customizability for a much more personal feel. 

My inspiration for this idea is coming from [Monkey Type](https://monkeytype.com/)
[Monkey Type GitHub](https://github.com/monkeytypegame/monkeytype)

Monkey Type is an open source typing test that includes a huge list of beautiful themes. 
I'll include some images so you can get a feel for what I have in mind.

This is a feature that I would really love to work on, but I want to make sure I implement it properly. I am not only new to this project, but I am new to contributing to projects in general. I was hoping that in this discussion post someone could point me in the right direction on this endeavor. 

![Screenshot_2023-03-01-00-40-22_3840x1080](https://user-images.githubusercontent.com/81405199/222055009-28f8de86-d833-4dd4-9cf2-8bfc23129296.png)
![Screenshot_2023-03-01-00-40-53_3840x1080](https://user-images.githubusercontent.com/81405199/222055011-b1336361-0ac3-4157-8780-131be31c1265.png)
![Screenshot_2023-03-01-00-41-25_3840x1080](https://user-images.githubusercontent.com/81405199/222055012-fb35190e-6d3a-44d5-8432-a881153a9859.png)
![Screenshot_2023-03-01-00-41-50_3840x1080](https://user-images.githubusercontent.com/81405199/222055014-9a2dc908-0362-41e8-a9c6-2cdaa066e8bb.png)
![Screenshot_2023-03-01-00-42-05_3840x1080](https://user-images.githubusercontent.com/81405199/222055015-44b003a8-1279-4fb3-bf96-4d8843d7ed3e.png)

**Comment by hyoban** [comment]
Need some time to design the theme system, currently, it is just a dark mode based on tailwind
